plugins/modules/zos_copy.py

```diff
@@ -2040,8 +2067,11 @@ def allocate_destination_data_set(
             size = int(src_attributes.total_space)
             record_format = src_attributes.recfm
             record_length = int(src_attributes.lrecl)
-
-            dest_params = get_data_set_attributes(dest, size, is_binary, record_format=record_format, record_length=record_length, type="PDSE", volume=volume)
+            if executable:
```

Comment:
I will test, but I remember a lot a of errors trying the copy to SEQ.

Reply:
Thanks for validating, so I think then since we know that copying an executable into a sequential data is not permitted , there is no point to create a LIBRARY, my thinking is just remove the is executable portion highlighted here:
<img width="960" alt="image" src="https://github.com/ansible-collections/ibm_zos_core/assets/25803172/717018f6-09b2-4f1f-92d4-bc18da661ecc">

Reply:
I think we also should do one more thing because I can not think of a use case we want to have the option `excutable:true` and the src/dest be a sequential data set. Thus there is no point on letting the FSUMF149 come up , we can fail early.
 
Thus we have this method called `is_compatable` we could update it to check this condition, if a user sets executable:true and src or dest are sequential data sets we can print out a nice error and maybe include this FSUM one as well. You can find the method [here](https://github.com/ansible-collections/ibm_zos_core/blob/4e7983e6ccaba382fd39b11e09429074afa5ccfc/plugins/modules/zos_copy.py#L1707)

If you look at the table, using -X with with sequential data sets will result in error as you have shown.
<img width="1223" alt="image" src="https://github.com/ansible-collections/ibm_zos_core/assets/25803172/e733c758-48d8-41d4-bc4d-b7c2bcc745de">

Reply:
`is_compatible` function was updated to reflect unsupported combination of sequential `src` or `dest` with `executable=true`. Marking this topic as 'resolved' since sequential data set support is out of scope for this iteration.